test/views/ValidBankEligibilityControllerViewSpec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import forms.EligibilityForm
+import org.jsoup.Jsoup
+import views.html.elgibility.{eligibility, validBank}
+
+
+class ValidBankEligibilityControllerViewSpec extends CommonViewSpecHelper with CharitiesViewMessages{
```

Comment:
As an aside, it looks like the CommonViewSpecHelper and the ControllerTestSpec can be compressed into a singular TestHelper.